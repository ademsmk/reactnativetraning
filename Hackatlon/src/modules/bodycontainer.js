import 'react-native';
import React from 'react';
import {View} from 'react-native';
import {TouchableOpacity, Image} from 'react-native';

export function BodyContainer() {
  return <View style={{flex: 6}}></View>;
}
