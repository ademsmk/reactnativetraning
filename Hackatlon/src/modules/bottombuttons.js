import React from 'react';
import {View, StyleSheet, Text, Image} from 'react-native';

export function BottomButtons() {
  return (
    <View
      style={{
        alignItems: 'stretch',
        flex: 1,
        flexDirection: 'row',
        marginBottom: 50,
      }}>
      <View
        style={{
          flex: 1,
          margin: 20,
          marginTop: 50,
        }}>
        <View
          accessible={true}
          focusable={false}
          isTVSelectable={true}
          onClick={() => {}}
          onResponderGrant={() => {}}
          onResponderMove={() => {}}
          onResponderRelease={() => {}}
          onResponderTerminate={() => {}}
          onResponderTerminationRequest={() => {}}
          onStartShouldSetResponder={() => {}}
          style={{
            opacity: 1,
          }}>
          <View
            style={{
              alignContent: 'center',
              alignItems: 'center',
              backgroundColor: 'white',
              borderWidth: 1,
              flexDirection: 'row',
              height: 50,
              justifyContent: 'center',
            }}>
            <Image
              source={{
                testUri: '../../../asset/arrow.png',
              }}
              style={{
                height: 20,
                marginRight: 5,
                width: 20,
              }}
            />
            <Text
              style={{
                color: 'black',
                fontFamily: 'Arial',
                fontSize: 25,
                fontWeight: '400',
                textAlign: 'center',
                textAlignVertical: 'center',
              }}>
              Start
            </Text>
          </View>
        </View>
      </View>
    </View>
  );
}

const styles = StyleSheet.create({
  image: {height: 30, width: 30, marginLeft: 20},
});
