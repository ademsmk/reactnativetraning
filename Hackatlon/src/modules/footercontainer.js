import React from 'react';
import {View, Text, Image} from 'react-native';

export function FooterContainer() {
  return (
    <View
      style={{
        flex: 5,
      }}>
      <View
        style={{
          height: 30,
          paddingBottom: 5,
          paddingLeft: 20,
        }}>
        <Text
          style={{
            color: 'white',
            fontFamily: 'Arial',
            fontSize: 20,
            fontWeight: '600',
          }}>
          Intensity...
        </Text>
      </View>
      <View
        style={{
          height: 30,
          paddingBottom: 5,
          paddingLeft: 20,
        }}>
        <Text
          style={{
            color: 'white',
            fontFamily: 'Arial',
            fontSize: 20,
            fontWeight: '600',
          }}>
          26 min | get In Shape
        </Text>
      </View>
      <View
        style={{
          height: 60,
          paddingBottom: 5,
          paddingLeft: 20,
          paddingTop: 10,
        }}>
        <Text
          style={{
            color: 'white',
            fontFamily: 'Arial',
            fontSize: 35,
            fontWeight: '600',
          }}>
          Full Body Flow
        </Text>
      </View>
      <View
        style={{
          alignItems: 'stretch',
          flex: 1,
          flexDirection: 'row',
          marginBottom: 50,
        }}>
        <View
          style={{
            flex: 1,
            margin: 20,
            marginTop: 50,
          }}>
          <View
            accessible={true}
            focusable={false}
            isTVSelectable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={{
              opacity: 1,
            }}>
            <View
              style={{
                alignContent: 'center',
                alignItems: 'center',
                backgroundColor: 'white',
                borderWidth: 1,
                flexDirection: 'row',
                height: 50,
                justifyContent: 'center',
              }}>
              <Image
                source={{
                  testUri: '../../../asset/arrow.png',
                }}
                style={{
                  height: 20,
                  marginRight: 5,
                  width: 20,
                }}
              />
              <Text
                style={{
                  color: 'black',
                  fontFamily: 'Arial',
                  fontSize: 25,
                  fontWeight: '400',
                  textAlign: 'center',
                  textAlignVertical: 'center',
                }}>
                Start
              </Text>
            </View>
          </View>
        </View>
      </View>
    </View>
  );
}
