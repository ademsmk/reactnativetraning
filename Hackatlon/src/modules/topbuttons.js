import 'react-native';
import React from 'react';
import {View} from 'react-native';
import {TouchableOpacity, Image} from 'react-native';
import {LoveIconButton} from '../components/iconbuttons/loveicon';
import {FileUploadIconButton} from '../components/iconbuttons/fileupload';
import {UserIconButton} from '../components/iconbuttons/usericon';

export function TopButtons() {
  return (
    <View
      style={{
        flex: 5,
        flexDirection: 'row',
        alignContent: 'flex-end',
        justifyContent: 'flex-end',
        marginRight: 40,
      }}>
      <LoveIconButton></LoveIconButton>
      <FileUploadIconButton></FileUploadIconButton>
      <UserIconButton></UserIconButton>
    </View>
  );
}
