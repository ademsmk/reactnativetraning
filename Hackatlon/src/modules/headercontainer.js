import 'react-native';
import React from 'react';
import {View} from 'react-native';
import {TouchableOpacity, Image} from 'react-native';
import {BackIconButton} from '../components/iconbuttons/backicon';
import {TopButtons} from './topbuttons';

export function HeaderContainer() {
  return (
    <View
      style={{
        flex: 1,
        marginTop: 20,
        flexDirection: 'row',
        alignContent: 'center',
        justifyContent: 'center',
      }}>
      <BackIconButton></BackIconButton>
      <TopButtons></TopButtons>
    </View>
  );
}
