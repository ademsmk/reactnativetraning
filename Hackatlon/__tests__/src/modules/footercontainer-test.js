import 'react-native';
import React from 'react';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

import { FooterContainer } from "../../../src/modules/footercontainer"; 

describe('FooterContainer -> ./src/modules/footercontainer', () => {
    it(' should render View component as root item', () => {
        const tree = renderer.create(<FooterContainer />).toJSON();
        expect(tree.type).toEqual("View");  
    }); 
    it(' should match snapshot', () => {
        const tree = renderer.create(<FooterContainer />).toJSON();
        expect(tree).toMatchSnapshot();
    });  
});
