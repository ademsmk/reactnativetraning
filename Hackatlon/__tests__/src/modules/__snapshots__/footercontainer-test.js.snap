// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterContainer -> ./src/modules/footercontainer  should match snapshot 1`] = `
<View
  style={
    Object {
      "flex": 5,
    }
  }
>
  <View
    style={
      Object {
        "height": 30,
        "paddingBottom": 5,
        "paddingLeft": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontFamily": "Arial",
          "fontSize": 20,
          "fontWeight": "600",
        }
      }
    >
      Intensity...
    </Text>
  </View>
  <View
    style={
      Object {
        "height": 30,
        "paddingBottom": 5,
        "paddingLeft": 20,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontFamily": "Arial",
          "fontSize": 20,
          "fontWeight": "600",
        }
      }
    >
      26 min | get In Shape
    </Text>
  </View>
  <View
    style={
      Object {
        "height": 60,
        "paddingBottom": 5,
        "paddingLeft": 20,
        "paddingTop": 10,
      }
    }
  >
    <Text
      style={
        Object {
          "color": "white",
          "fontFamily": "Arial",
          "fontSize": 35,
          "fontWeight": "600",
        }
      }
    >
      Full Body Flow
    </Text>
  </View>
  <View
    style={
      Object {
        "alignItems": "stretch",
        "flex": 1,
        "flexDirection": "row",
        "marginBottom": 50,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
          "margin": 20,
          "marginTop": 50,
        }
      }
    >
      <View
        accessible={true}
        focusable={false}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Object {
              "alignContent": "center",
              "alignItems": "center",
              "backgroundColor": "white",
              "borderWidth": 1,
              "flexDirection": "row",
              "height": 50,
              "justifyContent": "center",
            }
          }
        >
          <Image
            source={
              Object {
                "testUri": "../../../asset/arrow.png",
              }
            }
            style={
              Object {
                "height": 20,
                "marginRight": 5,
                "width": 20,
              }
            }
          />
          <Text
            style={
              Object {
                "color": "black",
                "fontFamily": "Arial",
                "fontSize": 25,
                "fontWeight": "400",
                "textAlign": "center",
                "textAlignVertical": "center",
              }
            }
          >
            Start
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
